lib/jormungandr/src/Cardano/Wallet/Jormungandr/Binary.hs

```diff
@@ -305,33 +305,29 @@ getMessage = label "getMessage" $ do
     -- corresponds to the txId (a.k.a "tx hash").
     fragId <- Hash . blake2b256 . BL.toStrict
         <$> lookAhead (getLazyByteString $ fromIntegral size)
-    msgType <- fromIntegral <$> getWord16le
+
+    -- A null byte for later extension
+    _nullByte <- getWord8
+    fragSpec <- getWord8
```

Comment:
While less correct, it would have worked the old way too?

Reply:
Well, they changed from `little-endian` to `big-endian`, so using a `getWord8Be` would have worked indeed. I just thought it'd be clearer this way as it better reflects the way they've written the `format.abnf` spec :+1:

Reply:
> they changed from little-endian to big-endian

Ah